website/blog/2024-06-20-run-ferretdb-cloudnativepg-kubernetes.md

```diff
@@ -0,0 +1,370 @@
+pod/postgres-cluster-2                     1/1     Running   0          10m
+pod/postgres-cluster-3                     1/1     Running   0          9m53s
+
+NAME                           TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)           AGE
+service/cnpg-webhook-service   ClusterIP   10.104.218.212   <none>        443/TCP           20m
+service/ferretdb-service       NodePort    10.107.244.213   <none>        27017:30987/TCP   3m55s
+service/postgres-cluster-r     ClusterIP   10.107.141.185   <none>        5432/TCP          17m
+service/postgres-cluster-ro    ClusterIP   10.110.7.253     <none>        5432/TCP          17m
+service/postgres-cluster-rw    ClusterIP   10.98.97.107     <none>        5432/TCP          17m
+
+NAME                                  TYPE                       DATA   AGE
+secret/cnpg-ca-secret                 Opaque                     2      17m
+secret/cnpg-webhook-cert              kubernetes.io/tls          2      17m
+secret/ferretuser-secret              kubernetes.io/basic-auth   2      19m
+secret/postgres-cluster-ca            Opaque                     2      17m
+secret/postgres-cluster-replication   kubernetes.io/tls          2      17m
+secret/postgres-cluster-server        kubernetes.io/tls          2      17m
+secret/sh.helm.release.v1.cnpg.v1     helm.sh/release.v1         1      20m
+```
+
+### Connect to FerretDB via `mongosh`
+
+Create a temporary `mongosh` client to connect to the FerretDB instance.
+
+```sh
+kubectl -n cnpg run mongosh-client --image=rtsp/mongosh --rm -it -- bash
+```
+
+That will start up a `mongosh-client` within the namespace.
+
+```sh
+mongosh 'mongodb://ferretuser:<password>@ferretdb-service:27017/ferretdb?authMechanism=PLAIN'
+```
+
+Run the following command to get the `ferretuser` password:
+
+```sh
+kubectl get secret ferretuser-secret -n cnpg -o jsonpath='{.data.password}' | base64 --decode
+```
+
+This should connect you to the FerretDB instance and the output should look like this:
+
+```text
+mongosh-client:/# mongosh 'mongodb://ferretuser:<password>@ferretdb-service:27017/ferretdb?authMechanism=PLAIN'
+Current Mongosh Log ID: 667388855d643f01696bade7
+Connecting to:    mongodb://<credentials>@ferretdb-service:27017/ferretdb?authMechanism=PLAIN&directConnection=true&appName=mongosh+2.2.9
+Using MongoDB:    7.0.42
+Using Mongosh:    2.2.9
+For mongosh info see: https://docs.mongodb.com/mongodb-shell/
+------
+   The server generated these startup warnings when booting
+   2024-06-20T01:40:21.558Z: Powered by FerretDB v1.21.0 and PostgreSQL 16.2.
+   2024-06-20T01:40:21.558Z: Please star us on GitHub: https://github.com/FerretDB/FerretDB.
+   2024-06-20T01:40:21.558Z: The telemetry state is undecided.
+   2024-06-20T01:40:21.558Z: Read more about FerretDB telemetry and how to opt out at https://beacon.ferretdb.com.
+------
+ferretdb>
+```
+
+### Test CRUD commands in `mongosh`
+
+```json5
```

Comment:
Let's follow the guide https://github.com/FerretDB/FerretDB/blob/main/website/docs/contributing/writing-guide.md#code-blocks

Reply:
@Fashander the guide indicates request should be `js` block and response should be `json5`. Why not follow what it says?

Reply:
The reason for following the guide is the request should be reformatted to make sure the consistency. The response should be as is from the mongosh.